sphinxcontrib/qthelp/__init__.py

```diff
@@ -86,6 +86,11 @@ def get_theme_config(self) -> Tuple[str, Dict]:
         return self.config.qthelp_theme, self.config.qthelp_theme_options
 
     def handle_finish(self) -> None:
+        if self.config.qthelp_basename != self.config.project:
```

Comment:
I think this condition is difficult to understand because we need to know the relationship between `config.project` and `config.qthelp_basename`. How about overriding `self.epilog` always?

Reply:
Ok, done.